examples/ecs/system_welding.rs

```diff
@@ -1,23 +1,29 @@
+/// System welding is a valuable but niche tool that allows you to pass the output from one system
+/// as input into a second system on a one-to-one basis.
+///
+/// It is most useful for error handling, or applying similar system adaptors to existing code.
+/// For more general-purpose system communication, you should probably use Events instead.
+/// For system ordering, use `.before` and `.after`.
 use anyhow::Result;
 use bevy::prelude::*;
 
 fn main() {
```

Comment:
All other examples (with the exception of the removal-detection one) with comments that are placed before `main`, have those after the Imports.
For consistency, I would do the same here.

```suggestion
use anyhow::Result;
use bevy::prelude::*;

/// System welding is a valuable but niche tool that allows you to pass the output from one system
/// as input into a second system on a one-to-one basis.
///
/// It is most useful for error handling, or applying similar system adaptors to existing code.
/// For more general-purpose system communication, you should probably use Events instead.
/// For system ordering, use `.before` and `.after`.
fn main() {
```